libkbfs/data_types.go

```diff
@@ -242,11 +298,19 @@ func (p BlockPointer) IsValid() bool {
 	return true
 }
 
+// IsDirect returns true if it is known for sure that the pointed-to
+// block is a direct block.
+func (p BlockPointer) IsDirect() bool {
```

Comment:
I think `IsDirect` implies a boolean flag, and it's easy to forget that `false` may mean two things. Mind removing this and just having `== directBlock` checks?